extensions/amp-ad-exit/0.1/config.js

```diff
@@ -15,6 +15,7 @@
  */
 
 import {user} from '../../../src/log';
```

Comment:
Ignore changes to this file - it was changed in #10714 and will be rebased away.